docs/modules/reference/pages/provisioning/policies.adoc

```diff
@@ -10,6 +10,19 @@ For information on how to extend the functionality of these policies, see the xr
 == Common policy parameters
 
 The matchBehavior action is a boolean control that determines how the optional parameters will be evaluated.
-Setting this parameter’s value to `ALL_PARAMETERS` causes Provisiond to evaluate each optional parameter with boolean AND logic and the value `ANY_PARAMETERS` causes OR logic to be applied.
+
+ALL_PARAMETERS::
+    All defined parameters must match (AND Boolean logic) for the policy to apply.
+ANY_PARAMETERS::
+    Any defined parameters can match (OR Boolean logic) for the policy to apply.
+NO_PARAMETERS::
+    No defined parameters can match (opposite of ANY_PARAMETERS) for the policy to apply.
 
 String values are assumed to be a substring match, unless the parameter is prefixed with `~` to indicate the policy should evaluate as a regular expression.
```

Comment:
In this case, are we saying that any defined String parameters are looking for a substring match in other parameters by default? If so, I'd suggest the following:

```suggestion
String parameters are set to search for a substring match by default. If you want the policy to evaluate a String as a regular expression, prefix it with `~`.
```

Reply:
```suggestion
String parameters are set to search for a substring match by default.
If you want the policy to evaluate a parameter against a regular expression, prefix the value with `~`.
```